core-feature-pack/common/src/main/resources/content/bin/systemd/README

```diff
@@ -0,0 +1,80 @@
+= How to configure WildFly as a Systemd service
+
+The following steps describe the process to configure WildFly as a Systemd service.
+You can use the provided wildfly-[standalone,domain].service file as a template for your own Systemd Unit File.
+It can be adjusted to your needs, for example, you can manually change the user that runs the service,
+the location of the WildFly installation, logs, etc.
+Alternatively, you can use the generate_systemd_unit.sh script to automatically generate a new Systemd Unit File
+using this server installation as the WildFly home.
+
+If you want to install WildFly as a Systemd service from scratch as a Systemd service, follow the steps below.
+
+== Install WildFly
+
+    # unzip -d /opt wildfly-34.0.0.Final.zip
+    # ln -s /opt/wildfly-34.0.0.Final /opt/wildfly
```

Comment:
I need to check but not sure if that would be fine taking into account that this file contains some other linux commands, I don't want to make them be replaced by maven applying environment properties across this file

Reply:
That is a good point. I don't see any others with the pattern `${property.name}`, but it's definitely something to consider since that is the linux way.

I was mostly trying to think of a way where we don't have to keep updating the version, but maybe it doesn't matter.